api/app/signals/apps/users/v1/views/__init__.py

```diff
@@ -0,0 +1,5 @@
+from signals.apps.users.v1.views.users import UserViewSet
+
+__all__ = (
+    'UserViewSet',
+)
```

Comment:
Noted earlier, but: add all viewsets in `users` app under V1.

Reply:
Will do